### PR TITLE
SLR Support

### PR DESCRIPTION
**Pull request by lucatume**

### CODE REVIEW

**Review by bordoni — DISMISSED** [review]
I wonder which branch we should use, I would urge this to be released with the plugin consolidation. So probably `bucket/plugin-consolidation`

File: src/Tribe/Assets.php [hunk 1]
```diff
@@ -34,17 +25,54 @@ public static function instance() {
 	 * Register the Methods in the correct places.
 	 *
 	 * @since 4.3
+	 * @since TBD Emptied of all hooks in favor fo the stellarwp/assets library.
```
Comment (Camwyn) [inline src/Tribe/Assets.php:28]:
```suggestion
	 * @since TBD Emptied of all hooks in favor of the stellarwp/assets library.
```

**Review by Camwyn — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "tribe-common",
-  "version": "5.2.7",
+  "version": "5.4.0-dev",
```
Comment (stratease) [inline package.json:3]: We skipped 5.3?

File: src/functions/template-tags/general.php [hunk 3]
```diff
@@ -559,14 +564,14 @@ function tribe_format_currency( $cost, $post_id = null, $currency_symbol = null,
 	 *
 	 * @category Events
 	 * @param  string $optionName Name of the option to retrieve.
-	 * @param string  $default    Value to return if no such option is found.
+	 * @param string $default    Value to return if no such option is found.
```
Comment (stratease) [inline src/functions/template-tags/general.php:567]: Can you fix the rest of the docblock spacing, since you started?

**Review by stratease — CHANGES_REQUESTED** [review]

**Pull request merged**